src/popup/components/AddressTruncated.vue

```diff
@@ -33,12 +33,19 @@ export default defineComponent({
 <style lang="scss" scoped>
 @use '../../styles/typography';
 
-.dots {
-  @extend %face-mono-15-regular;
+.address-truncated {
+  @extend %face-mono-12-medium;
 
-  letter-spacing: -0.25em;
-  text-align: center;
-  margin-left: -1px;
-  margin-right: 3px;
+  letter-spacing: 0.07em;
+
+  .dots {
+    @extend %face-mono-16-regular;
+
+    letter-spacing: -0.25em;
+    text-align: center;
+    margin-left: -1px;
+    margin-right: 3px;
+  }
 }
+
```

Comment:
Were the styles changed everywhere where the component is used?

Reply:
because component had wrong font

Reply:
It's likely but it's used in many places and styles may vary

Reply:
yes, and still, this component had before everywhere wrong font. I fix this problem here